### ISSUE TITLE
One of this symbols broke service

### ISSUE THREAD

**Issue by qWici** [dawnlabs/carbon#504]
**Describe the bug**
Doesnt save image if code contains this string:
```
possibleSymbols += '~`!@#$%^&*()_+{}"|?><|-=[]/.,'
```

**To Reproduce**
1. Go to website
2. In field for code paste
```
possibleSymbols += '~`!@#$%^&*()_+{}"|?><|-=[]/.,'
```
3. Choose Javascript language
4. Try to save PNG or SVG

![Screenshot](https://user-images.githubusercontent.com/11472929/45905770-35fc4900-bdfa-11e8-8049-ba0b1e4187cc.png)

**Info (please complete the following information):**
 - OS [e.g. macOS, Linux, Windows, iOS]: Windows 7
 - Browser [e.g. chrome, safari]: Chrome 69
 - 
<details>
  <summary>Code Snippet (If Applicable)</summary>
possibleSymbols += '~`!@#$%^&*()_+{}"|?><|-=[]/.,'
</details>

**Comment by qWici** [comment]
@mfix22 I'm sorry. Choose Javascript language

**Comment by AbdulazizAlswaji** [comment]
Remove the percentage sign and it should work just fine.

**Comment by mfix22** [comment]
Yeah we have had many issues in the past due to percentage signs breaking due to a bad encoding. I would like to fix this once-and-for-all

**Comment by mfix22** [comment]
Closed with #568
